### PR TITLE
DM-1974: render line breaks for practice summary

### PR DESCRIPTION
**Pull request by ahay-agile6**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-1974

## Description - what does this code do?
renders line breaks in the practice summary

## Testing done - how did you test it/steps on how can another person can test it 
1. go to any practice edit page (i.e. http://localhost:3200/practices/preventing-contextual-errors-pce-program/edit/introduction)
2. In the "Summary" make some new lines and returns in the textarea
3. Save
4. Go to the show page (http://localhost:3200/practices/preventing-contextual-errors-pce-program)
5. should render the line breaks as you put them in the PE

## Screenshots, Gifs, Videos from application (if applicable)
image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/practices/show/introduction/_introduction.html.erb [hunk 1]
```diff
@@ -63,7 +63,7 @@
       </div>
       <%# practice summary %>
       <p class="desktop:grid-col-10 grid-col-12 margin-bottom-2 font-sans-lg line-height-sans-6">
-        <%= @practice.summary %>
+        <%= safe_join(h(@practice.summary).split("\r\n"), tag(:br)) %>
```
Comment (ahay-agile6) [inline app/views/practices/show/introduction/_introduction.html.erb:66]: let me know if you guys think this should be a named method... like... `render_line_breaks(string)`
i thought it was simple enough to place as is, but i'm open to thoughts :smile:

Comment (emyl3) [reply app/views/practices/show/introduction/_introduction.html.erb:66]: this feels good for now... maybe if we are using it more frequently we could make a method for it 🤔

Comment (bradjohnson92008) [reply app/views/practices/show/introduction/_introduction.html.erb:66]: Yeah - i think it's fine as is.  Should we wrap all text area elements with this going forward?

Comment (ahay-agile6) [reply app/views/practices/show/introduction/_introduction.html.erb:66]: @bradjohnson92008 no, not yet.
when alex requests them, which may be soon, hah.

**Pull request merged**